feature/experimental/bgp/otg_tests/link_bandwidth_test/metadata.textproto

```diff
@@ -0,0 +1,23 @@
+# proto-file: third_party/openconfig/featureprofiles/proto/metadata.proto
```

Comment:
Update the proto-file path here to github.com/openconfig/featureprofiles/proto/metadata.proto.

Reply:
fixed